src/views/order_list_view.js

```diff
@@ -0,0 +1,91 @@
+import Backbone from 'backbone';
+import _ from 'underscore';
+import $ from 'jquery';
+
+import Order from '../models/order';
+import OrderView from '../views/order_view';
+
+import Quote from '../models/quote';
+import QuoteList from '../collections/quote_list';
+
+const OrderListView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.quoteList = params.symbols
+    this.bus = params.bus;
+    this.listenTo(this.model, 'update', this.render);
+    this.orderForm(params.symbols.models);
+  },
+
+  render() {
+    this.$('#orders').empty();
+    this.model.each((order) => {
+      const orderView = new OrderView({
+        model: order,
+        template: this.template,
+        tagName: 'li',
+        className: 'order',
+        bus: this.bus,
+      });
+      this.$('#orders').append(orderView.render().$el);
+    });
+    return this;
+  },
+  orderForm(symbol_names) {
+    symbol_names.forEach( (symbol_name) => {
+      $('select').append(`<option>${symbol_name.get('symbol')}</option>`)
+    });
+  },
+  events: {
+    'click button.btn-buy': 'buyOrder',
+    'click button.btn-sell': 'sellOrder'
+  },
+  buyOrder: function(event) {
+    event.preventDefault();
+    const orderData = {};
+    orderData['symbol'] = this.$(`[name=symbol]`).val();
```

Comment:
This function and `sellOrder` are almost identical. Could you consolidate the two?